quattor/aii/server/config.pan

```diff
@@ -11,8 +11,18 @@ variable AII_DHCP_DHCPD_CONF_FILE ?= '/etc/dhcpd.conf';
 
 include 'components/aiiserver/config';
 
-# aii-shellfe configuration
+# Enable tftpd (on EL9 only for backward compatibility)
+include 'components/systemd/config';
+'/software/components/systemd/unit/tftp' = {
+    if ( OS_VERSION_PARAMS['majorversion'] >= '9' ) {
+        SELF['type'] = 'socket';
+        SELF['state'] = 'enabled';
+        SELF['startstop'] = true;
+    };
+    SELF;
```

Comment:
I believe this will result in the path being set to undef when line 17 is false. I can't remember if having an `else null` works here, or whether that leaves `/software/components/systemd/unit` in the tree which might be a problem on RH6; although maybe RH6 support is OK to drop?

Reply:
RH6 support has been dropped already in 24.10...

Reply:
But you were perfectly right that the current code doesn't work on EL7/8 as it left a configuration path undefined... This is the reason the `test_templates` was failing! Fixed now, thanks for spotting it!